src/app/components/button/button.ts

```diff
@@ -214,7 +214,9 @@ export class ButtonDirective implements AfterViewInit, OnDestroy {
     }
 
     getIconClass() {
-        return this.loading ? 'p-button-loading-icon ' + (this.loadingIcon ? this.loadingIcon : 'p-icon') : this._icon;
+        return this.loading
+            ? 'p-button-loading-icon ' + (this.loadingIcon ? 'pi-spin ' + this.loadingIcon : 'pi pi-spin pi-spinner')
```

Comment:
it might be too much to add pi-spin to the custom loading icon. 
one might have it's own spinning logic on the outside

Reply:
mmm I don't understand your point. How it's the way that you say?

Thanks for the review!

Reply:
You added the class 'pi-spin' to the loadingIcon (`this.loadingIcon ? 'pi-spin ' + this.loadingIcon`), but this add the default spinning to the custom icons. 
This might not be wanted, as one might have a different spinning login for it's own custom icon, an might not want the default one from the default primeng spinner. It should be the user that passes 'pi-spin ' with the rest of the string in the loadingIcon input.

Reply:
Thanks again @GiacomoDM, so you recommend remove pi-spin and leave the user write when he choose the icon?
Maybe @cetincakiroglu can orient us how to proceed in your both comments.